pkg/tracer/tracer_linux_test.go

```diff
@@ -226,6 +226,8 @@ func TestCloseInFlightTCPConnectionWithEBPFWithData(t *testing.T) {
 	// Explicitly close this TCP connection
 	c.Close()
 
+	time.Sleep(50 * time.Millisecond)
```

Comment:
Why do we need the `time.Sleep` here?

Reply:
This test fails too often. Looks like sometimes it can't process "closing of connection" in time before we request it.
The sleep is a bit ugly, but it definitely works, and the amount of sleep is small